pyproject.toml

```diff
@@ -3,18 +3,19 @@ name = "scicookie"
 version = "0.6.2"  # semantic-release
 description = "Cookiecutter template for a Python package"
 authors = ["Ivan Ogasawara <ivan.ogasawara@gmail.com>"]
-license = "BSD"
+license = "BSD-3-Clause"
 include = [
     {path = "src/scicookie/cookiecutter.json"},
     {path = "src/scicookie/{{cookiecutter.project_slug}}"},
     {path = "src/scicookie/hooks"},
+    {path = "src/scicookie/profiles"},
 ]
 
 [tool.poetry.scripts]
 "scicookie" = "scicookie.__main__:app"
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<4"
+python = ">=3.8.1,<3.12"
```

Comment:
Is there a reason why we're not supporting Python 3.12?

Reply:
not exactly, but we would need to test it first on CI
but the other deps we have like makim and sugar, for example, is not working with 3.12
we can add support for that in a follow-up
but I think it is better to add support for python version incrementally instead of <4, because it could silently break